doc/CCLUTMOCKING.md

```diff
@@ -212,6 +212,17 @@ call cclutRemoveAllMockImplementations(null)
 7. The table mocking APIs are not supported when called from within a reportwriter section.  It might be tempting to use a dummyt query to set up mock data from a record structure, but various mocking calls such as cclutCreateMockTable, cclutRemoveMockTable and cclutAddMockData cannot be executed within the context of a query (because the implementations execute queries). Use a for loop instead.
  
 8. Mocking the tdbexecute "reply_to" entity is unsupported under certain conditions, specifically if a call to free the "reply_to" entity is made just prior to calling tdbexecute.  If the scenario is truly necessary for a test, the best alternative is to define and use a subroutine for freeing the "reply_to" entity within the script and use a mock for that subroutine which does not actually perform the freeing of the "reply_to" entity.
+
+9. In versions of CCL prior to 8.14.4 (32-bit) and 9.2.4 (64-bit), mocking is not supported in situations where all of the following are true:
```

Comment:
The details are a rather distracting. Couldn't this simply be `Mock substitutions can fail to occur with older versions of CCL if the script under test executes some other CCL program before declaring/implementing its subroutines (32-bit < 8.14.1 and 64-bit < 9.2.4).`

Sure they might not fail if ten other things are not also true, but the problem and remedy are now obvious. Don't execute other programs before declaring subroutines.